tests/api/task/conditional_dependencies/test_logging_utils.py

```diff
@@ -0,0 +1,321 @@
+from unittest.mock import Mock
+
+import pytest
+
+from fides.api.task.conditional_dependencies.logging_utils import (
+    _format_condition_list,
+    _format_evaluation_message,
+    _format_group_condition,
+    _format_leaf_condition,
+    _format_leaf_condition_message,
+    _is_group_condition,
+    _is_leaf_condition,
+    format_evaluation_failure_message,
+    format_evaluation_success_message,
+)
+
+
+def create_leaf_mock(field_address: str, operator: str, expected_value=None):
+    """Create a properly configured leaf condition mock"""
+    mock = Mock(spec=[])
+    mock.field_address = field_address
+    mock.operator = operator
+    mock.expected_value = expected_value
+    return mock
+
+
+def create_group_mock(logical_operator: str, condition_results=None):
+    """Create a properly configured group condition mock"""
+    if condition_results is None:
+        condition_results = []
+
+    mock = Mock(spec=[])
+    mock.logical_operator = logical_operator
```

Comment:
`logical_operator` is a `GroupOperator enum`. Could you add a focused test to make sure `_is_group_condition` and `_is_leaf_condition` correctly recognize enum-based values (e.g., `GroupOperator.and_`, `Operator.gte`) in addition to plain strings?”